src/auto-complete-element.js

```diff
@@ -64,6 +64,18 @@ export default class AutocompleteElement extends HTMLElement {
     }
   }
 
+  get fetchWhenBlank(): boolean {
+    return this.hasAttribute('fetch-when-blank')
```

Comment:
I'm not too convinced about the attribute name. How about `allow-blank-query`?

What does the rest of the team think?